### ISSUE TITLE
dpms toggle turns display off, but doesn't turn display on

### ISSUE THREAD

**Issue by subnut** [swaywm/sway#6416]
### Please fill out the following:
- **Sway Version:**
  ```
  sway version 1.6.1
  ```

- **Debug Log:**
  https://gist.github.com/subnut/c9af5ae4afe6775bab2d969183d87b84

- **Configuration File:**
  ```
  exec_always kitty
  ```
  (kitty is my terminal emulator)

- **Stack Trace:**
  - This is only needed if sway crashes.
  Sway doesn't crash, not needed

- **Description:**
  - The steps you took in plain English to reproduce the problem.

- Two monitors, `eDP-1` and `HDMI-A-1`
- Run `swaymsg output HDMI-A-1 dpms toggle`, it works (display turns off)
- Run `swaymsg output HDMI-A-1 dpms toggle`, it **doesn't work** (display **doesn't** turn on)
- Exit sway

**Comment by emersion** [comment]
`dpms toggle` support was added in https://github.com/swaywm/sway/pull/6099, but isn't in any release yet.

**Comment by subnut** [comment]
@emersion  
But #6099 was merged on Mar 25, and the latest release is on Jun 24 

I don't understand 😕

**Comment by emersion** [comment]
The latest release only contains bug fixes.
